scripts/utils/filesystem.mjs

```diff
@@ -0,0 +1,28 @@
+// @ts-check
+import * as nodefs from "node:fs";
+
+const CP_R_OPTIONS = { force: true, recursive: true };
+const MKDIR_P_OPTIONS = { recursive: true, mode: 0o755 };
+const RM_R_OPTIONS = { force: true, maxRetries: 3, recursive: true };
```

Comment:
Curious, have you found that you actually need the retries?

Reply:
I see it comes from previous uses

Reply:
I think this is mostly intended for Windows where a file can be locked.